storybookjs/storybook
# Provide styling hook for Addon Info story body

Issue: While most parts of the info addon ui offer configurable styles, the container in which the story body is rendered is unstylable.

## What I did
I added an empty styling hook for the story body, used in both the inline and standalone views.

## How to test
Configure a test to style the story body and observe its effects.

```
addWithInfo('default', () => <div>Fantastic</div>, {
  styles: (stylesheet) => Object.assign(stylesheet, {
    infoStory: {
      backgroundColor: '#4267B2'
    }
  })
});
```
